src/WebJobs.Script/Binding/DocumentDBBinding.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    internal class DocumentDBBinding : FunctionBinding
+    {
+        private BindingDirection _bindingDirection;
+
+        public DocumentDBBinding(ScriptHostConfiguration config, string name, string databaseName, string collectionName, bool createIfNotExists, FileAccess access, BindingDirection direction) :
+            base(config, name, "documentdb", access, false)
+        {
+            this.DatabaseName = databaseName;
+            this.CollectionName = collectionName;
+            this.CreateIfNotExists = createIfNotExists;
```

Comment:
Yeah, each codebase has its preferred patterns, and the all the WebJobs repos do not use this. Makes sense to be consistent.